study-builder/studies/snippets/text-question-address-required.conf

```diff
@@ -17,10 +17,18 @@
           {
```

Comment:
Similarly, this file is shared with other studies. Maybe better to create separate one for atcp.

Reply:
Fixed